src/h3lib/lib/geoCoord.c

```diff
@@ -47,7 +47,7 @@ double _posAngleRads(double rads) {
  * @return Whether or not the two coordinates are within the threshold distance
  *         of each other.
  */
-bool geoAlmostEqualThreshold(const GeoCoord* p1, const GeoCoord* p2,
+bool geoAlmostEqualThreshold(const GeoCoord *p1, const GeoCoord *p2,
```

Comment:
aside: strange that this formatting change got included

Reply:
That is the correct formatting tho, right? That's what the formatter results in for me. I'm not sure why it is getting caught here and wasn't corrected earlier.

Reply:
Probably the formatter doesn't run on unedited files? No one's touched this in a bit.

Side note: Should update the copyright to `-2020`

Reply:
updated copyright